lib/matplotlib/quiver.py

```diff
@@ -188,7 +193,7 @@
 
 Returns
 -------
-matplotlib.quiver.Quiver
+quiver : `~matplotlib.quiver.Quiver`
```

Comment:
Can you just put the quoted type name here?

Reply:
Yes. I thought there were some issues in numpydoc with that, but it seems to work now.